documentation/api/command/v1/commands.markdown

```diff
@@ -32,19 +32,24 @@ The entire command MUST be encoded as UTF-8.
 
 ## Command submission
 
-Commands are submitted via HTTP to the `/pdb/cmd/v1` URL and must
-conform to the following rules:
+Commands must be submitted via HTTP to the `/pdb/cmd/v1` endpoint via one of
+two mechanisms:
 
-* A `POST` is used
-* The `POST` body must contain the JSON payload.
-* There is an `Accept` header that matches `application/json`.
-* The content-type is `application/json`.
+* Payload and parameters: This method entails POSTing the certname, command name,
+command version, and optionally the checksum as parameters, with the POST body
+containing the given command's wire format. This mechanism allows PuppetDB to
+provide some information at the wire without inspecting the command payload
```

Comment:
This mechanism allows PuppetDB to provide better validation and feedback at the time of POSTing without...